mysql/const.go

```diff
@@ -425,14 +429,22 @@ const (
 	ModePadCharToFullLength
 )
 
-// GetSQLMode gets the sql mode for string literal.
-func GetSQLMode(str string) SQLMode {
-	str = strings.ToUpper(str)
-	mode, ok := Str2SQLMode[str]
-	if !ok {
-		return ModeNone
+// GetSQLMode gets the sql mode for string literal. SQL_mode is a list of different modes separated by commas.
+func GetSQLMode(s string) (SQLMode, error) {
+	strs := strings.Split(s, ",")
+	if strings.Count(strs[len(strs)-1], " ") == len(strs[len(strs)-1]) {
```

Comment:
How about just `strings.TrimRight(s, " ")`?

Reply:
yep. That's exactly what need to do. And it's clearer.